packages/core/framework/src/config/types.ts

```diff
@@ -87,6 +87,24 @@ export type AdminOptions = {
    * ```
    */
   backendUrl?: string
+  /**
+   * The URL of your Medusa storefront application. This URL is used as a prefix to some
+   * links in the admin that require performing actions in the storefront. For example,
+   * this URL is used as a prefix to shareable payment links for orders with
+   * outstanding amounts.
+   *
+   * @example
+   * ```js title="medusa-config.js"
+   * module.exports = defineConfig({
+   *   admin: {
+   *     storefrontUrl: process.env.MEDUSA_STOREFRONT_URL ||
+   *       "http://localhost:9000"
```

Comment:
nit: to keep it inline with our starter

```suggestion
   *       "http://localhost:8000"
```